target/linux/ath79/image/generic.mk

```diff
@@ -171,6 +171,24 @@ define Device/comfast_cf-e120a-v3
 endef
 TARGET_DEVICES += comfast_cf-e120a-v3
 
+define Device/comfast_cf-wr650ac-v1
+  ATH_SOC := qca9558
+  DEVICE_TITLE := COMFAST CF-WR650AC v1
+  DEVICE_PACKAGES := kmod-ath9k kmod-ath10k-ct ath10k-firmware-qca988x-ct kmod-usb2 kmod-usb-ledtrig-usbport kmod-leds-reset kmod-mdio-gpio kmod-swconfig kmod-fs-squashfs kmod-leds-gpio kmod-ledtrig-gpio kmod-ledtrig-netdev kmod-rfkill om-watchdog
+  IMAGE/sysupgrade.bin := append-kernel | pad-to 1536k | append-rootfs | pad-rootfs | check-size $$$$(IMAGE_SIZE)
+  IMAGE_SIZE := 16128k
+endef
+TARGET_DEVICES += comfast_cf-wr650ac-v1
+
+define Device/comfast_cf-wr650ac-v2
+  ATH_SOC := qca9558
+  DEVICE_TITLE := COMFAST CF-WR650AC v2
```

Comment:
Please update to use DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT as introduced in:
https://github.com/openwrt/openwrt/commit/8c3dbaf8b49233e275b2dce4e30dc0a2bf8c86c5

Reply:
fixed